collector_bmc.go

```diff
@@ -63,11 +63,17 @@ func (c BMCCollector) Collect(result freeipmi.Result, ch chan<- prometheus.Metri
 		logger.Debug("Failed to parse bmc-info data", "target", targetName(target.host), "error", err)
 		systemFirmwareVersion = "N/A"
 	}
+	bmcUrl, err := freeipmi.GetBMCInfoBmcUrl(result)
+	if err != nil {
+		// This one is not always available.
+		logger.Debug("Failed to parse bmc-info data", "target", targetName(target.host), "error", err)
+		systemFirmwareVersion = "N/A"
```

Comment:
This should set `bmcUrl`, not `systemFirmwareVersion`

Reply:
Thank you! Fixed